__tests__/setup/dataexplorer.test.js

```diff
@@ -79,66 +79,4 @@ describe('Data explorer', () => {
         let descriptionElement = document.querySelector(".data-category__h3");
         expect(descriptionElement).toBeVisible();
     })
-
-    test('Subcategory is not shown if all the indicators are qualitative', () => {
-        let config = new SAConfig();
-
-        const controller = new Controller(this, null, config, 1);
-        const dataMapperMenu = new DataMapperMenu(this);
-
-        configureDataExplorerEvents(controller, dataMapperMenu);
-
-        const payload = JSON.parse(JSON.stringify(quantQualBasePayload));
-        payload["Demo category"].subcategories["Demo subcategory"].indicators["Qualitative indicator"].content_type = "html";
-        payload["Demo category"].subcategories["Demo subcategory"].indicators["Qualitative indicator"].dataset_content_type = "qualitative";
-
-        controller.state = {
-            profile: {
-                geometries: {
-                    children: {
-                        "province": {
-                            "type": "FeatureCollection",
-                            "features": []
-                        }
-                    }
-                }
-            }
-        }
-
-        controller.triggerEvent("versions.indicators.ready", payload);
-
-        let subcategoryElement = document.querySelector(".data-category__h2");
-        expect(subcategoryElement).not.toBeVisible();
-    })
-
-    test('Excluded indicators are not shown', () => {
-        let config = new SAConfig();
-
-        const controller = new Controller(this, null, config, 1);
-        const dataMapperMenu = new DataMapperMenu(this);
-
-        configureDataExplorerEvents(controller, dataMapperMenu);
-        const payload = JSON.parse(JSON.stringify(quantQualBasePayload));
-        payload["Demo category"].subcategories["Demo subcategory"].indicators["Qualitative indicator"].chart_configuration = {
-            "exclude": ["data mapper"]
-        }
-
-        controller.state = {
-            profile: {
-                geometries: {
-                    children: {
-                        "province": {
-                            "type": "FeatureCollection",
-                            "features": []
-                        }
-                    }
-                }
-            }
-        }
-
-        controller.triggerEvent("versions.indicators.ready", payload);
-
-        let descriptionElement = document.querySelector(".data-category__h3");
-        expect(descriptionElement).not.toBeVisible();
-    })
```

Comment:
yes - since we need the summary endpoint not send these indicators at all, we now have these checks in the backend. you can see the slack convo also

https://openupsa.slack.com/archives/C024AJAJ3T8/p1661264111404579